### DISCUSSION TITLE
Incompatibility with Laravel 11

### DISCUSSION THREAD

**Discussion by tarikkamat** [trovee/laravel-repository#15] [Show and tell]
I ran `composer require trovee/laravel-repository`

./composer.json has been updated
Running composer update trovee/laravel-repository
Loading composer repositories with package information
Updating dependencies

Your requirements could not be resolved to an installable set of packages.

 Problem 1
    - Root composer.json requires trovee/laravel-repository * -> satisfiable by trovee/laravel-repository[v1.0.0].
    - trovee/laravel-repository v1.0.0 requires illuminate/contracts ^10.48 -> found illuminate/contracts[v10.48.0, ..., v10.48.7] but these were not loaded, likely because it conflicts with another require.

You can also try re-running composer require with an explicit version constraint, e.g. "composer require trovee/laravel-repository:*" to figure out if any version is installable, or "composer require trovee/laravel-repository:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

**Comment by uutkukorkmaz** [comment]
Hi @tarikkamat,

Thanks for the contribution. I actually planning adding Laravel 11 support at the package's 2.x version because that's a breaking change in my perspective. The issue #7 is all about that. 

Unfortunately, I have very little time for take care all of the package's needs and very desperate to see some pull requests. I've created the version branch and draft a release to the respective version.
